<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCompetitie</class>
 <widget class="QDialog" name="AddCompetitie">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(170, 170, 127);</string>
  </property>
  <widget class="QPushButton" name="b_back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>51</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(247, 247, 182);</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="denumire">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(247, 247, 182);
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="locatie">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(247, 247, 182);</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="puncte_participare">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(247, 247, 182);
</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="puncte_runda1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(247, 247, 182);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>107</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Locatie</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>35</y>
     <width>68</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Denumire</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Puncte pentru participare</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Puncte pentru runda 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Add_competitie">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>290</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(247, 247, 182);</string>
   </property>
   <property name="text">
    <string>Adauga</string>
   </property>
  </widget>
  <widget class="QLabel" name="picture">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>201</width>
     <height>121</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>250</width>
     <height>250</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Puncte pentru semifinala</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="puncte_semifinala">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(247, 247, 182);</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="puncte_finala">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(247, 247, 182);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Puncte pentru finala</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
